modules/EsetTools.py

```
@@ -1,7 +1,10 @@
 from .EmailAPIs import *
```

Ruff (0.8.0)

`from .EmailAPIs import *` used; unable to detect undefined names

(F403)